examples/test.py

```diff
@@ -0,0 +1,117 @@
+import hydra
+
+from omegaconf import DictConfig
+from omegaconf import OmegaConf
+from typing import List
+
+from enum import Enum
+from dataclasses import dataclass, field
+
+#@dataclass
+class NeMoConfig(dict):
+    """ Abstract NeMo Configuration class"""
+    pass
+    def serialize(self):
```

Comment:
Confusing what these serialize/deserialize functions suppose to do.
Are these "from_dict" and "to_dict" ? And then trivially read/write those dict using FileIO. APIs?

Reply:
Please note that in NeMo 0.11 I have introduced **serialize()/deserialize()** methods as those are **standard names** for functions covering that functionality.

Thus in my opinion are less confusing than yours "from_dict/from_config_dict" etc.

A matter of taste.